semantic-conventions/src/opentelemetry/semconv/main.py

```diff
@@ -51,12 +51,13 @@ def exclude_file_list(folder: str, pattern: str) -> List[str]:
     return file_names
 
 
-def filter_semconv(semconv, type_filter):
-    if type_filter:
+def filter_semconv(semconv, types_filter):
+    if types_filter:
+        types = types_filter.replace(" ", "").split(",")
```

Comment:
What is the goal of replacing spaces here? Maybe you wanted to do this?

```suggestion
        types = [t.strip() for t in types_filter.split(",")]
```